src/ui/components/UrlExportSettings.tsx

```diff
@@ -38,6 +38,7 @@ export const UrlExportSettings = () => {
   const { settings, updateSettings } = useContext<{settings: Settings, updateSettings: any}>(SettingsContext)
   const { tokens, setTokens } = useContext(TokenContext)
   const { figmaUIApi } = useContext(FigmaContext)
+  let message = '';
```

Comment:
We need to stay in react-land with this:

```js
const [commitMessage, setCommitMessage] = useState("")
```

Reply:
No problem. Sorry, I've never used React before now.

Reply:
No worries at all. The problem, why I mentioned it is, that without using useState, the react component does not refresh. This means without this change you could not type in the field at all.